CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.6.0
+* Enhancement - Add support for retention policy update of existing log groups (#121)
```

Comment:
reads a bit weirdly: how about: "Add support for updating the retention policy of existing log groups"

Reply:
yeah.. I agree. I will update it